packages/modeling/src/operations/booleans/scission.js

```diff
@@ -0,0 +1,43 @@
+const flatten = require('../../utils/flatten')
+
+// const geom2 = require('../../geometries/geom2')
```

Comment:
Hmm... is scission of 2D geometries possible?

Reply:
It's probably possible, but not straight forward.

Should it be implemented now? Not sure.